python-client/api-reference/usersession.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: UserSession
+---
+
+Checkout the documentation about [UserSessions](/concepts/usersession) for more information
+
+## Creating a UserSession
+
+```python
+session = await client.api.create_user_session(
+    participant_identifier="<PARTICIPANT_UUID>",
+    is_interactive=True,
+    metadata={"key": "value"}
+)
+```
+
+<ParamField path="is_interactive" type="boolean">
+  Has the user interacted with the app
+</ParamField>
+
+<ParamField path="started_at" type="string">
+  Start datetime
+</ParamField>
+
+<ParamField path="ended_at" type="string">
+  End of session datetime
+</ParamField>
+
+<ParamField path="participant_identifier" type="uuid">
+  Identifier of the participant
+</ParamField>
+
+<ParamField path="anon_participant_identifier" type="string">
+  Anonymous user
+</ParamField>
+
+<ParamField path="metadata" type="dict">
+  Metadata
```

Comment:
need more details (what is this useful for?)

Reply:
I don't really know what we can add here.